spec/controllers/tutor_controller.spec.rb

```diff
@@ -0,0 +1,195 @@
+require 'rails_helper'
+
+RSpec.describe TutorsController, type: :controller do
+  describe 'endpoints' do
+    before do
+      sign_in_auth
+    end
+
+    describe 'GET #index' do
+      it 'populates an array of all tutors' do
+        tutors = [create(:tutor)]
+        get :index
+        expect([assigns(:tutors).last]).to eq(tutors)
+      end
+
+      it 'renders the :index view' do
+        get :index
+
+        expect(response).to render_template :index
+      end
+    end
+
+    describe 'GET #show' do
+      before do
+        @tutor = create(:tutor)
+      end
+
+      it 'populates the specified tutor' do
+        get :show, id: @tutor
+        expect(assigns(:tutor)).to eq(@tutor)
+      end
+
+      it 'populates the coordinator of the specified tutor' do
+        job = VolunteerJob.where(tutor_id: @tutor.id).take
+        coordinator = job.nil? ? nil : Coordinator.find(job.coordinator_id)
```

Comment:
Why the conditional? The test should know whether there will or won't be a coordinator.